sdk/core/azure-core-amqp/src/amqp/session.cpp

```diff
@@ -238,8 +238,9 @@ namespace Azure { namespace Core { namespace Amqp { namespace _detail {
       m_claimsBasedSecurity = std::make_shared<ClaimsBasedSecurityImpl>(shared_from_this());
     }
     auto accessToken = GetConnection()->GetSecurityToken(audience, context);
-    Log::Stream(Logger::Level::Informational)
-        << "Authenticate with audience: " << audience << ", token: " << accessToken;
+
+    //    Log::Stream(Logger::Level::Verbose)
+    //        << "Authenticate with audience: " << audience << ", token: " << accessToken;
```

Comment:
Clean up commented out code (keep the logging?)

Reply:
Removed.